src/components/implementation/no_interface/llboot/boot_deps.h

```diff
@@ -636,6 +484,88 @@ boot_comp_thds_init(void)
 	assert(llboot->init_thd >= 0);
 }
 
+static inline void
```

Comment:
This is a stop-gap measure.  We need to just properly implement the mapping facilities so that map and populate in booter.c actually work.  Hard-coding the booting of the memmgr is only acceptable for this phase.  We need to generalize to get everything booting next.  We need to support the existing eco-system next.